src/libsyncengine/jobs/network/login/abstractloginjob.cpp

```diff
@@ -71,7 +71,26 @@ bool AbstractLoginJob::handleError(std::istream &is, const Poco::URI &uri) {
     try {
         jsonError = jsonParser.parse(is).extract<Poco::JSON::Object::Ptr>();
     } catch (Poco::Exception &exc) {
-        LOG_DEBUG(_logger, "Reply " << jobId() << " received doesn't contain a valid JSON error: " << exc.displayText().c_str());
+        LOG_WARN(_logger, "Reply " << jobId() << " received doesn't contain a valid JSON error: " << exc.displayText().c_str());
+
+        // Try to parse as string
+        std::stringstream errorStream;
```

Comment:
Is there a potential re-use this error formatting code? If so, it could be wrapped into some utils method.

Reply:
Agreed, I'll look into it

Reply:
Done